src/QsCompiler/Tests.Compiler/SyntaxTests.fs

```diff
@@ -32,18 +32,26 @@ let ``String parser tests`` () =
     |> List.iter (fun s -> Assert.True(simpleParseString (stringLiteral .>> eof) s, "Failed to parse string " + s))
 
     // testing whether tabs etc in strings are processed correctly
+    // Note: the below strings use triple-quote syntax where the
```

Comment:
This comment here no longer seems to be accurate.

Reply:
Whoops! I forgot to remove that when I went back to standard escaped strings.